process/block/preprocess/interfaces.go

```diff
@@ -33,3 +35,14 @@ type BlockSizeThrottler interface {
 	GetCurrentMaxSize() uint32
 	IsInterfaceNil() bool
 }
+
+// BalanceComputationHandler defines the functionality for addresses balances computation
```

Comment:
might add: ...used in preventing executing too many debit transactions after the proposer executed a credit transaction on the same account in the same block

Reply:
done